src/main/java/net/glowstone/command/CommandUtils.java

```diff
@@ -132,4 +129,22 @@ public static GlowWorld getWorld(CommandSender sender) {
         }
         return null;
     }
+
+    public static String getName(CommandSender sender) {
+        if (sender instanceof Entity) {
+            return getName((Entity) sender);
+        }
+        return sender.getName();
+    }
+
+    public static String getName(Entity entity) {
+        String name = entity.getName();
+        if (name == null || name.isEmpty()) {
+            name = entity.getType().getName();
+        }
+        if (entity.getCustomName() != null && !entity.getCustomName().isEmpty()) {
```

Comment:
People can name an entity as an empty string, right? I think it would be better to show `type:name`